### DISCUSSION TITLE
🙏 🙏Persisted Query By Using GET !!!

### DISCUSSION THREAD

**Discussion by brightchip** [Akryum/vue-apollo#1091] [Q&A]
My app uses Nuxt and The persisted queries seem work But I can't make it use **GET** instead of POST 
The app keep requesting with Post which is meaningless as It won't cache the requests to CDN

\[Official Link](https://www.apollographql.com/docs/apollo-server/performance/apq/)

`  

        clientConfigs: {
            default: {
                httpEndpoint: process.env.apolloServer,
                persisting: true,
                useGETForHashedQueries: true
            },
        },

        defaultOptions: {
            // See 'apollo' definition
            // For example: default query options
            $query: {
                loadingKey: 'loading',
                fetchPolicy: 'cache-and-network',

            },
            persisting: true,
            useGETForHashedQueries: true
        },

    },`

**Comment by lance-anderson** [comment]
@RailgunWinjoy I am running into this as well. Did you find a solution?

**Reply by lance-anderson** [reply]
I found this tip helpful https://github.com/nuxt-community/apollo-module/issues/365#issuecomment-733029476
